### DISCUSSION TITLE
Setting up a local webots.cloud *with competitions*

### DISCUSSION THREAD

**Discussion by riggas-ionio** [cyberbotics/webots-cloud#147] [Q&A]
Hi!..
we've used robotbenchmark for several years now in the Robotics & Programming course of an MSc class at Ionian University (https://di.ionio.gr/pmsdigappinnov/en/course-description/).
Sadly, free and open resources seem not to be available anymore; we could setup our own server, but I'd like to be sure what we end up with. 
I would need an environment like `https://webots.cloud/competition` **with the available competitions**.
What services do I need to setup for this? Is `https://cyberbotics.com/doc/guide/simulation-server` enough?
Thanks, Dimitrios

**Comment by omichel** [comment]
Yes, that should be sufficient. You may find more information in the wiki pages of this repository:
https://github.com/cyberbotics/webots-cloud/wiki (see pages on the right margin).
